README.md

```diff
@@ -755,6 +755,15 @@ You can also see the help for the command in the CLI:
 $ affinidi verify-vc --help
 ```
 
+### **affinidi start**
```

Comment:
Please put this command in alphabetical order to follow what has been done so far in the ReadMe file

Reply:
Not sure exactly how/what I should do ?
All the commands are not in alphabetical order.
Should I reorganize ?
Regarding what has been done, don't we have a release note or smt similar to this?

@dendavidov-affinidi @JonasEtzold

Reply:
I'll leave it as is it now in my PR as I don't exactly understand why/how I should change the alphabetical order here.